src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/CngKey.StandardProperties.cs

```diff
@@ -192,6 +192,23 @@ public int KeySize
                     throw errorCode.ToCryptographicException();
                 }
 
+                // The platform crypto provider always returns "0" for EC keys as of Windows 11 22H2.
```

Comment:
Is this a Windows bug?

Reply:
Well, it's certainly unexpected. I'm re-reading my comment and realizing it may not be conveying what I hoped it would. Windows has always behaved this way with the Platform Crypto Provider, and I most recently tested it in Windows 11 H2.

Reply:
Yeah, I re-worded that to be more clear (hopefully).